test/test-server-parser.js

```diff
@@ -199,15 +199,16 @@ var tests = [
                                       0x01,
                                       0x00,
                                       0x04,
-                                      0xFF, 0xFE, 0xFD, 0xFC, 0xFB, 0xFA,
-                                       0xF9, 0xF8, 0xF7, 0xF6, 0xF5, 0xF4,
-                                       0xF3, 0xF2, 0xF1, 0xF0,
+                                      0xFF, 0xFE, 0xE0, 0xD0,
+                                      0x00, 0x0C, 0x00, 0xA0,
+                                      0x00, 0x00, 0x03, 0x00,
+                                      0x00, 0x02, 0xB0, 0x01,
```

Comment:
Can you keep lines 203-205 indented? The reason is it makes it easier to visually scan and see that all of those bytes are part of the same field.